py3status/i3status.py

```diff
@@ -10,18 +10,168 @@
 from syslog import syslog, LOG_ERR, LOG_INFO
 from tempfile import NamedTemporaryFile
 from threading import Thread
+from time import time
 
 from py3status.profiling import profile
 from py3status.helpers import jsonify, print_line
 from py3status.events import IOPoller
 
 
+class I3statusModule:
+    """
+    This a wrapper for i3status items so that they mirror some of the methods
+    of the Module class.  It also helps encapsulate the auto time updating for
+    `time` and `tztime`.
+    """
+
+    def __init__(self, module_name, py3statusWrapper):
+        self.module_name = module_name
+        self.i3status = py3statusWrapper.i3status_thread
+        self.is_time_module = module_name.split()[0] in ['time', 'tztime']
+        self.item = {}
+        self.py3statusWrapper = py3statusWrapper
```

Comment:
If I'm not mistaken, we call it `py3_wrapper` everywhere else ? I'm not a fan of those camelCase like stuff.